ts-framework/create-function/gram-template-mcp/README.md

```diff
@@ -0,0 +1,43 @@
+# Gram Function MCP Template
+
+This template allows you to use the official [MCP TypeScript SDK][mcp-ts] to
+build and deploy [Gram Functions](https://getgram.ai).
+
+[mcp-ts]: https://github.com/modelcontextprotocol/typescript-sdk
+
+Gram Functions are tools and resources for LLMs and MCP servers that can do
+arbitrary tasks such as fetching data from APIs, performing calculations, or
+interacting with hosted databases.
+
+## Prerequisites
+
+- [Node.js](https://nodejs.org) version 22.18.0 or later
+- [Gram CLI](https://www.speakeasy.com/docs/gram/command-line/use)
+
+## Getting Started
+
+To get started, install dependencies and run the development server:
+
+```bash
+pnpm install
+```
```

Comment:
> To get started, install dependencies and run the development server:

The bash snippet doesn't include the dev server command.

```suggestion
To get started, install dependencies and run the development server:

```bash
pnpm install
pnpm run dev
```
```